collector/collector.go

```diff
@@ -73,26 +75,27 @@ type AttunityCollector struct {
 // NewAttunityCollector returns a pointer to an attunityCollector object
 // which implements the Prometheus Collector interface.
 // It should be registered to a Prometheus Registry.
-func NewAttunityCollector(cfg *Config) *AttunityCollector {
+func NewAttunityCollector(cfg *Config, ignoreCert bool) *AttunityCollector {
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: ignoreCert}
```

Comment:
I don't think that this line would actually work? I'm a little confused as to why you have to type cast it.

Reply:
Maybe consider setting the `Transport` field of the `client` in here with the TLS option: https://github.com/wbh1/attunity_exporter/blob/6b2367ee774cd372f9d5206035acf61fc7c1c8c1/collector/collector.go#L80-L84

Then change this bit to just modify `client.Transport.TLSNextProto` instead of creating a new `http.Transport` and assigning it. https://github.com/wbh1/attunity_exporter/blob/6b2367ee774cd372f9d5206035acf61fc7c1c8c1/collector/collector.go#L86-L94